### PR TITLE
Remove ThingScreenlet and APIO Consumer

### PR DESCRIPTION
**Pull request by victorlaerte**

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/util/extensions/ViewGroup+.kt [hunk 1]
```diff
@@ -1,22 +1,30 @@
 /**
  * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
+ * <p>
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/util/extensions/ViewGroup+.kt]: you can disable this in idea

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt [hunk 2]
```diff
@@ -177,12 +172,12 @@ class DDMFormView @JvmOverloads constructor(
...
 
 	override fun refreshVisibleFields() {
-		getInstantiatedPages().flatMap {
-			it.childrenSequence().asIterable()
-		}.mapNotNull {
-			it as? DDLFieldViewModel<*>
-		}.forEach {
-			it.refresh()
+		getInstantiatedPages().forEach { viewGroup ->
+			viewGroup.forEachChild { view ->
```
Comment (phcp) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt]: 🏆 🏆 🏆

File: android/library/src/main/java/com/liferay/mobile/screens/util/AndroidUtil.kt [hunk 3]
```diff
@@ -110,7 +111,18 @@ class AndroidUtil {
 
 		@JvmStatic
 		fun updateViewState(view: View, enabled: Boolean) {
-			view?.isEnabled = enabled
+			view.isEnabled = enabled
```
[the remark this answers is not in the document]
Comment (victorlaerte) [reply android/library/src/main/java/com/liferay/mobile/screens/util/AndroidUtil.kt]: If so we should use `View?` instead `View` in the signature. I'll fix that.

File: android/library/src/main/java/com/liferay/mobile/screens/ddm/form/service/openapi/GetFormServiceOpenAPI.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import org.json.JSONObject
+
+/**
+ * @author Victor Oliveira
+ */
+class GetFormServiceOpenAPI : GetFormService {
+	override fun getForm(formInstanceId: Long, serverUrl: String, onSuccess: (FormInstance) -> Unit,
+		onError: (Throwable) -> Unit) {
+
+		val formInstanceJSONStr = AndroidUtil.assetJSONFile("formInstance.json", LiferayScreensContext.getContext())
```
[the remark this answers is not in the document]
Comment (victorlaerte) [reply android/library/src/main/java/com/liferay/mobile/screens/ddm/form/service/openapi/GetFormServiceOpenAPI.kt]: np, this is a temporary method

**Pull request merged**